lib/index.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+// external modules
+const Assembler = require('stream-json/utils/Assembler');
+const assert = require('assert-plus');
+const JSONParseStream = require('stream-json/Combo');
+const through2 = require('through2');
+const stringifyStreamFactory = require('json-stream-stringify');
+
+
+/**
+ * create a JSON.parse that uses a stream interface. the underlying stream-json
+ * module used to do the parsing is actually a combo stream that has 3 internal
```

Comment:
what do u mean by combo? It's multiplexing 3 streams together, or is it 3 streams piped in serial?

Reply:
I'm just using nomenclature of the underlying module, but it is actually a combined three streams in one to avoid the overhead of streams themselves.